runtime/syntax/c.vim

```diff
@@ -287,7 +287,10 @@ syn keyword	cType		signed unsigned float double
 if !exists("c_no_ansi") || exists("c_ansi_typedefs")
   syn keyword   cType		size_t ssize_t off_t wchar_t ptrdiff_t sig_atomic_t fpos_t
   syn keyword   cType		clock_t time_t va_list jmp_buf FILE DIR div_t ldiv_t
-  syn keyword   cType		mbstate_t wctrans_t wint_t wctype_t
+endif
+if !exists("c_no_c95") " Amendment 1 to ISO C90
+  syn keyword cType             wint_t mbstate_t wctype_t wctrans_t
+  syn keyword cOperator		and bitor or xor compl bitand and_eq or_eq xor_eq not not_eq
```

Comment:
It's safe to say that almost no projects use iso646.h in C. Such keywords should be opt-in (off by default, turned on by variable definition).

```suggestion
  if exists("c_c95_iso646") " iso646.h macros
    syn keyword	cOperator	and bitor or xor compl bitand and_eq or_eq xor_eq not not_eq
  endif
```